### DISCUSSION TITLE
HEVC encoded MP4 video cut at keyframes the same way as AVC H.264 MP4 video files?

### DISCUSSION THREAD

**Discussion by pstein** [mifi/lossless-cut#1341] [Q&A]
Just a question about cutting MP4 video files which are encoded in the new HEVC codec:

Can I cut them in the same way as previous AVC H.264 video files?
In particular: Is there jump to next/prev keyframe handling the same?

Are there any differences in handling in LC between them?

**Comment by mifi** [comment]
LosslessCut doesn't natively support HEVC in the same way as H264, so preview will be a bit slow/low quality. but other than that it should work the same, and the results will still be lossless. Keyframes work similarly. Smart cut might not work as well for HEVC as H264 though. YMMV

**Comment by pstein** [comment]
Thank you for the answer.

Preview quality and speed is not important for me.

But what do you mean with "Smart Cut"?

Is the lack of smart cut for HEVC due to HEVC codec restrictions or due to missing LC implementation?

If the latter: Could you add it in one of the next LC versions?

**Reply by mifi** [reply]
> But what do you mean with "Smart Cut"?

see https://github.com/mifi/lossless-cut/issues/126

> Is the lack of smart cut for HEVC due to HEVC codec restrictions or due to missing LC implementation?

LLC supports smart cut with HEVC but because smart cut is experimental, hevc might not always work with smart cut. I personally had more success with h264, although some hevc files do work too
